### ISSUE TITLE
I can not clone release 2.3.4 using BuGet

### ISSUE THREAD

**Issue by asafieh** [JKorf/Binance.Net#39]
I can not download release 2.3.4, I am using vs 2015 vers 14 update 3.
 can you share compiled DLL library ..
 
I have treid all NETFramework,Versions from 2.0 to 4.7.1 without any luck.

 below is the output log:
--------------------------------

GET https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4')
  GET https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/Packages(Id='Binance.Net',Version='2.3.4')
  OK https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4') 905ms
Error finding repository for 'https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/': An error occurred while retrieving package metadata for 'Binance.Net.2.3.4' from source 'Microsoft and .NET'.
  A task was canceled.
Resolved actions to install package 'Binance.Net.2.3.4'
  GET https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4')
  OK https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4') 1392ms
Install failed. Rolling back...
Package 'Binance.Net.2.3.4' does not exist in project 'BinanceBot'
Package 'Binance.Net.2.3.4' does not exist in folder 'c:\users\ashraf\documents\visual studio 2015\Projects\BinanceBot\packages'
Could not install package 'Binance.Net 2.3.4'. You are trying to install this package into a project that targets '.NETFramework,Version=v2.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
========== Finished ==========

**Comment by JKorf** [comment]
Hi, could you share the logging when trying to install in a project targeting .Net version 4.5 or higher? Lower isn't supported, and the error in this log is it can't install on a .Net framework 2.0 project which makes sense.

**Comment by asafieh** [comment]
Hi, 
  GET https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4')
  OK https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4') 689ms
Resolved actions to install package 'Binance.Net.2.3.4'
  GET https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4')
  OK https://www.nuget.org/api/v2/Packages(Id='Binance.Net',Version='2.3.4') 793ms
Install failed. Rolling back...
Package 'Binance.Net.2.3.4' does not exist in project 'BinanceBot'
Package 'Binance.Net.2.3.4' does not exist in folder 'c:\users\ashraf\documents\visual studio 2015\Projects\BinanceBot\packages'
Could not install package 'Binance.Net 2.3.4'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.7', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
========== Finished ==========

**Comment by asafieh** [comment]
Can you please tell me which Framework version have you used I found similar issue :
https://stackoverflow.com/questions/34463262/nugets-install-in-visual-studio-2015

**Comment by JKorf** [comment]
The package is targeting .NetStandard2.0, which is supported by .Net framework 4.5 and up.

Can you try updating your Nuget client?

Go to Tools->Extensions and Updates and select Updates->Visual Studio Gallery. Update the NuGet Package Manager extension. Then try to install the package after Visual Studio gets restarted.

**Comment by asafieh** [comment]
There is no update for NuGet version i have is 3.4.4.1321 .. i have attempted to reinstall it .. 
Please not  I am able to install Binance.Net.2.0.0 and below .

Belowo is the new log.

Attempting to gather dependency information for package 'Binance.Net.2.3.4' with respect to project 'BinanceBot', targeting '.NETFramework,Version=v4.6'
Attempting to resolve dependencies for package 'Binance.Net.2.3.4' with DependencyBehavior 'Lowest'
Resolving actions to install package 'Binance.Net.2.3.4'
Resolved actions to install package 'Binance.Net.2.3.4'
Install failed. Rolling back...
Package 'Binance.Net.2.3.4' does not exist in project 'BinanceBot'
Package 'Binance.Net.2.3.4' does not exist in folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Could not install package 'Binance.Net 2.3.4'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.6', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
========== Finished ==========

**Comment by asafieh** [comment]
I found newer version v3.6.0 on nuget.org which was not showing  Tools->Extensions and Updates 
I have installed it and resolved the issue it only works with framework version 4.7 

below is the log:

Attempting to gather dependency information for package 'Binance.Net.2.3.4' with respect to project 'BinanceBot', targeting '.NETFramework,Version=v4.7'
Gathering dependency information took 4.45 sec
Attempting to resolve dependencies for package 'Binance.Net.2.3.4' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'Binance.Net.2.3.4'
Resolved actions to install package 'Binance.Net.2.3.4'
Retrieving package 'SuperSocket.ClientEngine.Core 0.9.0' from 'nuget.org'.
Retrieving package 'WebSocket4Net 0.15.1' from 'nuget.org'.
Retrieving package 'Binance.Net 2.3.4' from 'nuget.org'.
Retrieving package 'System.Collections.Specialized 4.3.0' from 'nuget.org'.
Retrieving package 'System.IO 4.3.0' from 'nuget.org'.
Retrieving package 'System.Linq 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.NameResolution 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Sockets 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.Extensions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Runtime.InteropServices 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Encoding 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Primitives 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.Algorithms 4.3.0' from 'nuget.org'.
Retrieving package 'System.Security.Cryptography.X509Certificates 4.3.0' from 'nuget.org'.
Retrieving package 'System.Net.Security 4.3.0' from 'nuget.org'.
Retrieving package 'System.Text.RegularExpressions 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading 4.3.0' from 'nuget.org'.
Retrieving package 'System.Threading.Timer 4.3.0' from 'nuget.org'.
Retrieving package 'Newtonsoft.Json 10.0.3' from 'nuget.org'.
Adding package 'Newtonsoft.Json.10.0.3' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'Newtonsoft.Json.10.0.3' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'Newtonsoft.Json.10.0.3' to 'packages.config'
Executing script file 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages\Newtonsoft.Json.10.0.3\tools\install.ps1'...
Successfully installed 'Newtonsoft.Json 10.0.3' to BinanceBot
Adding package 'System.Collections.Specialized.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Collections.Specialized.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Collections.Specialized.4.3.0' to 'packages.config'
Successfully installed 'System.Collections.Specialized 4.3.0' to BinanceBot
Adding package 'System.IO.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.IO.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.IO.4.3.0' to 'packages.config'
Successfully installed 'System.IO 4.3.0' to BinanceBot
Adding package 'System.Linq.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Linq.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Linq.4.3.0' to 'packages.config'
Successfully installed 'System.Linq 4.3.0' to BinanceBot
Adding package 'System.Net.NameResolution.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Net.NameResolution.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Net.NameResolution.4.3.0' to 'packages.config'
Successfully installed 'System.Net.NameResolution 4.3.0' to BinanceBot
Adding package 'System.Net.Sockets.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Net.Sockets.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Net.Sockets.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Sockets 4.3.0' to BinanceBot
Adding package 'System.Runtime.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Runtime.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Runtime.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime 4.3.0' to BinanceBot
Adding package 'System.Runtime.Extensions.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Runtime.Extensions.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Runtime.Extensions.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.Extensions 4.3.0' to BinanceBot
Adding package 'System.Runtime.InteropServices.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Runtime.InteropServices.4.3.0' to 'packages.config'
Successfully installed 'System.Runtime.InteropServices 4.3.0' to BinanceBot
Adding package 'System.Security.Cryptography.Encoding.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.Encoding.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Encoding 4.3.0' to BinanceBot
Adding package 'System.Security.Cryptography.Primitives.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.Primitives.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.Primitives.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Primitives 4.3.0' to BinanceBot
Adding package 'System.Security.Cryptography.Algorithms.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.Algorithms.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.Algorithms.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.Algorithms 4.3.0' to BinanceBot
Adding package 'System.Security.Cryptography.X509Certificates.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Security.Cryptography.X509Certificates.4.3.0' to 'packages.config'
Successfully installed 'System.Security.Cryptography.X509Certificates 4.3.0' to BinanceBot
Adding package 'System.Net.Security.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Net.Security.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Net.Security.4.3.0' to 'packages.config'
Successfully installed 'System.Net.Security 4.3.0' to BinanceBot
Adding package 'SuperSocket.ClientEngine.Core.0.9.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'SuperSocket.ClientEngine.Core.0.9.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'SuperSocket.ClientEngine.Core.0.9.0' to 'packages.config'
Successfully installed 'SuperSocket.ClientEngine.Core 0.9.0' to BinanceBot
Adding package 'System.Text.RegularExpressions.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Text.RegularExpressions.4.3.0' to 'packages.config'
Successfully installed 'System.Text.RegularExpressions 4.3.0' to BinanceBot
Adding package 'System.Threading.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Threading.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Threading.4.3.0' to 'packages.config'
Successfully installed 'System.Threading 4.3.0' to BinanceBot
Adding package 'System.Threading.Timer.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Threading.Timer.4.3.0' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'System.Threading.Timer.4.3.0' to 'packages.config'
Successfully installed 'System.Threading.Timer 4.3.0' to BinanceBot
Adding package 'WebSocket4Net.0.15.1' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'WebSocket4Net.0.15.1' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'WebSocket4Net.0.15.1' to 'packages.config'
Successfully installed 'WebSocket4Net 0.15.1' to BinanceBot
Adding package 'Binance.Net.2.3.4' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'Binance.Net.2.3.4' to folder 'C:\Users\Ashraf\Documents\Visual Studio 2015\Projects\BinanceBot\packages'
Added package 'Binance.Net.2.3.4' to 'packages.config'
Successfully installed 'Binance.Net 2.3.4' to BinanceBot
Executing nuget actions took 21.12 sec
========== Finished ==========
Time Elapsed: 00:00:32.1645283


thank you for your prompt support.
